app/src/userpages/components/ProfilePage/profilePage.pcss

```diff
@@ -7,6 +7,7 @@
   z-index: 11;
   top: -4em;
   position: relative;
+  font-size: 14px;
```

Comment:
Use `rem` here?

Reply:
Hmm, this is coming from https://github.com/streamr-dev/streamr-platform/pull/188. Do we actually need this?